src/core/fallcurve/internal/vt_fc_signature_collection_settings.c

```diff
@@ -100,8 +99,7 @@ VT_UINT fc_signature_compute_collection_settings(
         return VT_ERROR;
     }
 
-    falltime_datapoints   = round((float)falltime_calib_test / (float)*sampling_interval_us);
-    *sampling_interval_us = round(((float)(*sampling_interval_us) * (float)falltime_datapoints) / (float)VT_FC_SAMPLE_LENGTH);
+    *sampling_interval_us = round((float)falltime_calib_test / (float)VT_FC_SAMPLE_LENGTH);
```

Comment:
This looks like a simplification of logic, is that right?

Reply:
yep simplification of logic, that's correct